### PR TITLE
added tests

### PR DESCRIPTION
**Pull request by abolla22**
Checklist
* [ ] Used a fork of the feedstock to propose changes
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

File: recipe/run_test.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    y = random.randint(0,5000)
+    assert 0 <= x <= 5000, "Failed to assert the random number from given range"
+    assert x != y, "Failed to generate Random numbers"
+
+    # Returns a random value from the list
+    list1 = [1, 2, 3, 4, 5, 6]
+    output_3 = random.choice(list1)
+    assert output_3 in list1, "Failed to assert the random number from list Returned value is not in the list "
+
+test_pycrptodome_pkg()
```
Comment (remkade) [inline recipe/run_test.py]:
```suggestion
test_pycryptodome_pkg()
```

File: recipe/run_test.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
+from Crypto.Random import random
+
+
+def test_pycrptodome_pkg():
```
Comment (remkade) [inline recipe/run_test.py]:
```suggestion
def test_pycryptodome_pkg():
```

File: recipe/run_test.py [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+    """
+    Tests to validate the pycrpto pkg.
+    NOTE: Given test validations will fail on python >3.8 version since pycrpto pkg discontinued to support
+    py>3.8 versions
```
Comment (remkade) [inline recipe/run_test.py]: These comments seem to apply to pycrypto, can you update them for pycryptodome?

**Review by remkade — CHANGES_REQUESTED** [review]
Looks good! Some minor changes.

**Comment by skupr-anaconda** [comment]
The license should be `license: BSD-2-Clause`, see https://github.com/Legrandin/pycryptodome/blob/v3.10.1/LICENSE.rst

And in `host`:
- Missing `wheel` package
- Missing `setuptools` package

Also, it would be nice to add in the test section:
```
  commands:
    - pip check
```

**Review by skupr-anaconda — APPROVED** [review]

**Review by remkade — COMMENTED** [review]
Looks good to me!

**Review by remkade — APPROVED** [review]

**Pull request merged**